index.Rmd

```diff
@@ -49,7 +49,7 @@ provinceNight <- read_excel("provinceNight.xls")
 
 
 ```{r,warning=FALSE,message=FALSE}
-ggplot(provinceNight) +
+ggplot(provinceNight) + 
```

Comment:
Lines of the provinces with low values are hard to distinguish. Would using the log y scale help?